python/GafferSceneUI/LightEditor.py

```diff
@@ -339,12 +366,24 @@ def __editSelectedCells( self, pathListing, quickBoolean = True ) :
 		inspectors = {}
 		inspections = []
 
+		with Gaffer.Context( self.getContext() ) as context :
+			lightSetMembers = self.__settingsNode["in"].set( "__lights" ).value
+
 		with Gaffer.Context( self.getContext() ) as context :
 			for selection, column in zip( pathListing.getSelection(), pathListing.getColumns() ) :
 				if not isinstance( column, _GafferSceneUI._LightEditorInspectorColumn ) :
 					continue
 				for pathString in selection.paths() :
 					path = GafferScene.ScenePlug.stringToPath( pathString )
+
+					if (
+						( column == self.__muteColumn or column == self.__soloColumn ) and
+						not ( lightSetMembers.match( path ) & (
+							IECore.PathMatcher.Result.ExactMatch | IECore.PathMatcher.Result.DescendantMatch
+						) )
+					) :
+						continue
```

Comment:
Interacting with the Mute and Solo columns for a light blocker results in the popup "The selected cells cannot be edited in the current edit scope", which may lead users to think they could potentially edit these somehow? 

Might be better to be more specific here if we can?

Reply:
Yes, that's a nice addition. Done in https://github.com/GafferHQ/gaffer/pull/5895/commits/e6cdc08aa9823bbd059156cebc9e3e8f95bd8b3c